src/Microsoft.OpenApi/Models/OpenApiExtensibleDictionary.cs

```diff
@@ -27,16 +27,16 @@ protected OpenApiExtensibleDictionary():this(null) { }
         /// <param name="dictionary">The generic dictionary.</param>
         /// <param name="extensions">The dictionary of <see cref="IOpenApiExtension"/>.</param>
         protected OpenApiExtensibleDictionary(
```

Comment:
wouldn't it be better to make the dictionary parameter non nullable and optional with a default value of `[]` (if that's possible) ?

Reply:
Yeah that's possible. However I didn't want to tamper with/change model data as it might have side effects during serialization